src/utils/getters/getFilteredExtEvents.ts

```diff
@@ -0,0 +1,17 @@
+import { CalendarInfo, ExtendedEvent } from "../../types.js";
+
+export function getFilteredExtEvents(filterStartDate: Date, calendarInfo: CalendarInfo): ExtendedEvent[] {
```

Comment:
Maybe you could just have `extEvents: ExtendedEvent[]` instead of `calendarInfo` as the argument since you're not using `calendarInfo` for anything else ?